src/app/index.js

```diff
@@ -618,6 +602,8 @@ export default yeoman.generators.Base.extend({
     if (this.options.hurry) {
       return;
     }
+    shelljs.exec('git add --all');
+    shelljs.exec('git commit -a -m "generator run"');
```

Comment:
I think so. It's kind of a fine line between convenience and imposition and I feel like this crosses it just slightly.

For example, when running against an existing project, we don't want to be creating commits with files that may need manual cleanup.